app/javascript/styles/mastodon-light/diff.scss

```diff
@@ -266,10 +266,10 @@ html {
 // Change the background colors of status__content__spoiler-link
 .reply-indicator__content .status__content__spoiler-link,
 .status__content .status__content__spoiler-link {
-  background: $ui-base-color;
+  background: $ui-base-color !important;
 
   &:hover {
-    background: lighten($ui-base-color, 4%);
+    background: lighten($ui-base-color, 4%) !important;
```

Comment:
This change does the trick, but I'd be wary of using `!important`. Instead, try adding `&:focus` in addition to `&:hover`, as far as I can see, the issue is caused by not redefining `&:focus` in this context.

Reply:
@ClearlyClaire Thanks, much better solution!